### DISCUSSION TITLE
Pan tilt gimbal

### DISCUSSION THREAD

**Discussion by vitsikon** [ysoldak/HeadTracker#29] [General]
Юрий, добрый день. Отличный хэдтрекер получился из XIAO, спасибо что поделились проектом!
Меня очень заинтересовала 2х осевая поворотка. Сейчас по понятным причинам купить на cult3d не получится. Могу у вас напрямую купить?

**Comment by ysoldak** [comment]
Да, можно, конечно. Напишите мне в Facebook? (если он не заблокирован у вас, я не в курсе, или тогда предложите как связаться по-другому)
https://www.facebook.com/ysoldak

**Reply by vitsikon** [reply]
Vitsinsky.kv@gmail.com 
Facebука не имею)
